jsonrpc/jsonrpc.go

```diff
@@ -46,18 +52,30 @@ func StartJSONRPC(
 	clientCtx client.Context,
 	jsonRPCConfig config.JSONRPCConfig,
 ) error {
+
+	tmWsClient := ConnectTmWS("http://127.0.0.1:26657", "/websocket", svrCtx.Logger) //TODO: add config for tmWS server
```

Comment:
@jasonsong0 
I think we should change variable name to `CometWS`. ConnectTmWS should also be changed to `ConnectCometWS`. 
Because we don't use the name `tendermint` anymore, we use `cometBFT`